newIDE/app/src/stories/componentStories/BuildProgressAndActions.stories.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+import * as React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import muiDecorator from '../ThemeDecorator';
+import paperDecorator from '../PaperDecorator';
+
+import BuildProgressAndActions from '../../Export/Builds/BuildProgressAndActions';
+import {
+  erroredCordovaBuild,
+  pendingCordovaBuild,
+  pendingElectronBuild,
+  completeCordovaBuild,
+  completeElectronBuild,
+  completeWebBuild,
+  game1,
+} from '../../fixtures/GDevelopServicesTestData';
+
+export default {
+  title: 'Export/BuildProgressAndActions',
+  component: BuildProgressAndActions,
+  decorators: [muiDecorator, paperDecorator],
+};
+
+export const Errored = () => (
+  <BuildProgressAndActions build={erroredCordovaBuild} game={game1} />
+);
+
+export const PendingElectronBuild = () => (
```

Comment:
I like the idea, but I've tried quickly and couldn't get something nice working 🤔 
I also need to change the prop publicWebBuildId, but only if it's a type web (didn't search long how to do that)

For the amount we use storybook, I'm leaving it like that :)

Reply:
Note: in an effort to help future developers or future us to discover all the use cases of a component, it might be worth having a bunch of different states explicitly in the stories. The control can be still useful to avoid an exponential number of cases, but I think the most important cases should be separate stories so that I can quickly browse them and look and say "ok nothing crashed and everything is looking good, so it should be ok".

In a more distant future where we do screenshots of important components, it will also be helpful to have these important use cases as separate stories (rather than getting the tool, like Chromatic, doing screenshots to change the controls).

Overall the idea is to find a balance between "not too much stories" and "not too much controls" :)

Reply:
There are 2 conflicting use-cases:

- As a dev, did I break something and you want to see all possible states rapidly only browsing the side menu
- As a designer, I want to make sure that all possibilities are covered so controls are more exhaustive

If we have a product designer, we'll see if they want to use the storybook as such